pysc2/lib/features.py

```diff
@@ -211,17 +212,41 @@ def __init__(self, game_info=None, screen_size_px=None, minimap_size_px=None,
     Raises:
       ValueError: if game_info is None and screen or minimap sizes are missing.
     """
+    self._map_size = None
     if game_info:
       fl_opts = game_info.options.feature_layer
       screen_size_px = point.Point.build(fl_opts.resolution)
       minimap_size_px = point.Point.build(fl_opts.minimap_resolution)
+      self._map_size = point.Point.build(game_info.start_raw.map_size)
+      self._camera_width_world_units = fl_opts.width
+      self._feature_layer_screen_size = point.Point.build(fl_opts.resolution)
+      self._world_to_screen = transform.Linear(point.Point(1, -1),
```

Comment:
If these are never used outside this function, they don't need to be instance variables and can just be locals.

Reply:
they are all used in `_update_camera` except `_screen_to_fl_screen`